server/handler.go

```diff
@@ -903,6 +903,59 @@ func (h *Handler) SetStoreLimitTTL(data string, value float64, ttl time.Duration
 	}, ttl)
 }
 
+// GetAllRequestHistroyHotRegion get historyHotRegions as request.
+func (h *Handler) GetAllRequestHistroyHotRegion(request *statistics.HistoryHotRegionsRequest) (*statistics.HistoryHotRegions, error) {
+	var hotRegionTypes []string
+	if len(request.HotRegionTypes) != 0 {
+		hotRegionTypes = request.HotRegionTypes
+	} else {
+		hotRegionTypes = cluster.HotRegionTypes
+	}
+	iter := h.s.hotRegionStorage.NewIterator(hotRegionTypes, request.StartTime, request.EndTime)
+	results := make([]*statistics.HistoryHotRegion, 0)
+	regionSet := make(map[uint64]bool)
+	storeSet := make(map[uint64]bool)
+	peerSet := make(map[uint64]bool)
+	for _, id := range request.RegionIDs {
+		regionSet[id] = true
```

Comment:
Why need request region IDs and peerIDs? I think only `StoresIds` and `KeyRange` are reasonable.